### DISCUSSION TITLE
Option to label gates with names ?

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#1157] [Q&A]
A useful addition to Digital would be to label gates with a name, that is inserted e.g. inside the gate symbol. Could that be implemented in a next version or is there a way to do this now ?

**Comment by hneemann** [comment]
It is possible to place text freely in the circuit. With this you can insert a label for the gates.

**Reply by jongena** [reply]
Thank you for the hint. This is indeed possible. However adding a label automatically (using a "label" as is done for other components)
would be a plus ... of course, if it does not require too much effort to implement it. 

Jef Ongena
